lib/ex_webrtc/dtls_transport.ex

```diff
@@ -108,6 +108,7 @@ defmodule ExWebRTC.DTLSTransport do
       ice_connected: false,
       buffered_local_packets: nil,
       buffered_remote_packets: nil,
+      buffered_remote_rtp_packets: [],
```

Comment:
We aren't sure if these are RTP packets, it might as well be RTCP (or even SCTP data if we account for it, see other comment), so the name is a bit misleading.

I was thinking of reusing the `buffered_remote_packets`, as it's used for packets before negotiation, then is flushed (affair), so it could be used for that, although this might make the code a bit more cryptic.

Reply:
Yeah, I was wary of reusing the same buffer because of that. If you have suggestions for alternative names, I'm all ears

Reply:
For now leave it, I'll think about it when taking care of buffering DTLS data.